factfinder/main.py

```diff
@@ -357,6 +357,17 @@ def calculate_e_m_p_z(self, pff_variable:str, geotype:str) -> pd.DataFrame:
         df = df.rename(columns=columns)
         return df[["census_geoid", "pff_variable", "geotype", "e", "m", "p", "z"]]
 
+    def calculate_poverty_p_z(self, v: Variable, geotype: str) -> pd.DataFrame:
+        """
+        For below poverty vars, the percent and percent MOE are taken from the ACS,
+        but they come from E and M fields, not PE and PM fields. This function
+        calculates the E and M for the associated percent variable, then renames as
+        P and Z to join with the count variable.
+        """
+        pct_df = calculate(f"{v.pff_variable}_pct", geotype=geotype)
```

Comment:
calculate -> calculate_e_m so that we can do all the rounding/cleaning by the end